hyco-https/lib/_hyco_outgoing.js

```diff
@@ -784,25 +802,24 @@ OutgoingMessage.prototype._flush = function _flush() {
 };
 
 OutgoingMessage.prototype._flushOutput = function _flushOutput(socket) {
-  var ret;
   var outputLength = this.output.length;
   if (outputLength <= 0)
-    return ret;
```

Comment:
>ret [](start = 11, length = 3)

Well that's weird (the previous code).

Reply:
ret = socket.send() actually returns null all the time, no idea why it was there

---
In reply to: [340264737](https://github.com/Azure/azure-relay-node/pull/44#discussion_r340264737) [](ancestors = 340264737)